test/diagnostics/no_warnings_as_errors.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %empty-directory(%t)
+// RUN: not %target-swift-frontend -typecheck -verify -warnings-as-errors -no-warnings-as-errors %s 2>&1 | %FileCheck %s
```

Comment:
1. Since we aren't producing any output files and saving them to `%t`, we shouldn't need `%empty-directory`.
2. I think you could replace `%target-swift-frontend -typecheck -verify` with `%target-typecheck-verify-swift`.
3. I'm not sure why there's a `not` and a `2>&1` here... Maybe I'm misunderstanding what this test is checking?

Reply:
👍 
Thanks for the pointer to `docs/Testing.md`!